File: src/main/java/pl/yalgrin/playnite/simplesync/repository/objects/GameDiffRepository.java
package pl.yalgrin.playnite.simplesync.repository.objects;

import org.springframework.data.r2dbc.repository.R2dbcRepository;
import org.springframework.stereotype.Repository;
import pl.yalgrin.playnite.simplesync.domain.objects.GameDiff;

@Repository
public interface GameDiffRepository extends R2dbcRepository<GameDiff, Long> {
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/repository/GameDiffRepository.kt
package pl.yalgrin.playnite.simplesync.library.repository

import org.springframework.data.r2dbc.repository.R2dbcRepository
import org.springframework.stereotype.Repository
import pl.yalgrin.playnite.simplesync.library.domain.GameDiff

@Repository
interface GameDiffRepository : R2dbcRepository<GameDiff, Long>